test/mocks/v2alpha1/kube/client/clientset/versioned/fake/register.go

```diff
@@ -29,7 +29,7 @@ import (
 
 var scheme = runtime.NewScheme()
 var codecs = serializer.NewCodecFactory(scheme)
-var parameterCodec = runtime.NewParameterCodec(scheme)
+
```

Comment:
why is this being removed? is this file even used?

Reply:
This is auto-generated by our clientset codegen: https://github.com/solo-io/solo-kit/blob/master/Makefile#L84. I would have to look further into how it is actually used in the solo-kit repo. This is updated because I updated the kube libraries we're using here since they were well behind our other projects. 

I think if we want to update/remove this clientset code, it should be done separately. Are you ok with the change as is, due to the library update?